contributing.md

```diff
@@ -0,0 +1,29 @@
+## Contributing
+
+We welcome contributions!
+
+We're using Github to write this guide as it's more likely to persist longer and allows for other people to discuss and contribute changes. It's a relatively new style of writing that most of us don't have experience with, so we're all experimenting on this together!
+
+A few tips on making good PRs:
+
+### Goals
+
+We'd like to strike a balance between accurate and complete information, and trying not to overwhelm the reader with too many things they need to do, or technicalities. There are going to be tradeoffs sometimes, but they're not completely at odds. If a section or file is growing too big with information, it might be worth factoring the details out into a separate section or file. Just like when a function grows too big!
+
+**Emphasis markers** help the reader focus on the key point of a section.
+
+> Blockquotes allow you to *reduce* emphasis on personal stories and more advanced details.
+
+### Consistency
+
+If you have something to add in a section, please read the whole section over and check that:
+
+- The text still follows a reasonable progression
+- The tone is similar
+- There are no major contradictions with what's already there (it'd confuse the reader!)
+
+That doesn't mean we don't want you to disagree with us! We just want to keep the guide as consistent as possible. Ways to disagree productively include opening an issue with proposed changes first, or writing the alternative point of view as a personal story.
+
+### On the use of pronouns
+
+Like a regular codebase, there's going to be some artifacts in the writing that we'll want to "refactor" over time. As the number of contributors increase, the usage of the pronoun "I" (which is still pretty common in the docs) might get confusing. Try to be mindful of that as you write!
```

Comment:
add something about either adding your name to show it's your own advice or story, or to not use "I" at all